data/mods/Magiclysm/field.json

```
@@ -43,6 +43,7 @@
   {
     "id": "fd_safe_shoppers",
     "type": "field_type",
+    "//": "context: `hair on your neck`, according to maleclypse, `I've always heard that when you suspect you are being watched it feels like the hair on your neck is lifting away from your body`. the effect of field is delete ",
     "intensity_levels": [
       {
         "name": "hair on your neck",
```

```suggestion
    "intensity_levels": [
      {
        "name": {
          "//~": "context: `hair on your neck`, according to maleclypse, `I've always heard that when you suspect you are being watched it feels like the hair on your neck is lifting away from your body`. the effect of field is delete ",
          "str": "hair on your neck"
        },
```

See https://github.com/CleverRaven/Cataclysm-DDA/pull/51680#discussion_r711481319